### PR TITLE
SNAP-2086: Pulse should display external tables in the list too

### PR DESCRIPTION
**Pull request by snappy-sachin**
## Changes proposed in this pull request

  - Code changes to capture external tables details and to display them on UI.

## Patch testing

  - Tested Manually

## ReleaseNotes.txt changes

  - Yes.

## Other PRs 

Store: https://github.com/SnappyDataInc/snappy-store/pull/357

### CODE REVIEW

File: core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala [hunk 1]
```diff
@@ -188,11 +189,28 @@ object SnappyEmbeddedTableStatsProviderService extends TableStatsProviderService
             asInstanceOf[java.util.ArrayList[SnappyRegionStatsCollectorResult]]
             .asScala
       }
+
+      // External Tables
+      val hiveTables: java.util.List[ExternalTableMetaData] =
```
Comment (rishitesh) [inline core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala]: Can be simplified. 
`val externalTables = Misc.getMemStore.getExternalCatalog.getHiveTables(true).asScala.filter(extTable => extTable.tableType.equalsIgnoreCase("EXTERNAL")).map(extTable => new SnappyExternalTableStats (extTable.entityName, extTable.tableType, extTable.provider, extTable.externalStore, extTable.dataSourcePath, extTable.driverClass))`

Comment (sumwale) [reply core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala]: Or one step more with collect :)

```scala
hiveTables.asScala.collect {
  case table if table.tableType.equalsIgnoreCase("EXTERNAL") =>
      new SnappyExternalTableStats(...)
}
```

Comment (snappy-sachin) [reply core/src/main/scala/io/snappydata/SnappyTableStatsProviderService.scala]: Updated code as per Sumedh's suggestion.

**Comment by snappy-sachin** [comment]
Feedback comments are taken care. Hence merging.

**Pull request merged**